ChangeLog

```diff
@@ -4,6 +4,10 @@ Pylint's ChangeLog
 What's New in Pylint 1.8?
 =========================
 
+	* Fixing u'' string in superfluous-parens message 
```

Comment:
please use spaces exclusively; also we need similar entry in whats_new/1.8.rst file